addons/dark-www/experimental_scratchr2.css

```
@@ -697,6 +697,9 @@ input.link.black {
   background-image: url("%addon-self-dir%/assets/meow.png");
   background-size: 20px 15px;
 }
+.stats li {
+  color: var(--darkWww-page-statsTextColor);
```

You can use the existing `box-scratchr2HeaderText` variable.

```suggestion
  color: var(--darkWww-box-scratchr2HeaderText);
```